tests/JasmineSidetreeErrorValidator.ts

```diff
@@ -50,4 +51,50 @@ export default class JasmineSidetreeErrorValidator {
       fail(`Expected error '${expectedSidetreeErrorCode}' did not occur.`);
     }
   }
+
+  /**
+   * Fails the current spec if the execution of the function does not throw the expected BitcoinError.
+   *
+   * @param functionToExcute The function to execute.
+   * @param expectedBitcoinErrorCode The expected Bitcoin error code.
+   */
+  public static expectBitcoinErrorToBeThrown (functionToExcute: () => any, expectedBitcoinErrorCode: string): void {

```

Comment:
Would Jasmine's expect(...).toThrow(...) be sufficient?

e.g.
```javascript
expect(() => { Operation.validateUpdatePayload(updatePayload); }).toThrow(new SidetreeError(ErrorCode.OperationUpdatePayloadMissingOrUnknownProperty));
```

Reply:
Jasmine's .toThrow(...) does a toString() to the error for comparison. And our errors sometimes also have a message appended which causes the comparison to fail. That was the original reason behind adding this utility class.